loleaflet/css/toolbar.css

```diff
@@ -1052,3 +1052,85 @@ menu-entry-with-icon.padding-left + menu-entry-icon.width */
 .menu-entry-no-icon {
 	padding-left: 42px;
 }
+
+.userInfo {
+	position: fixed;
+	z-index: 1050;
+	right: 4px;
+	top: 0px;
+	height: 38px;
+	background-color: transparent;
+	display: flex !important;
+	min-width: 150px;
+	align-items: center;
+}
+
+.userInfo-content {
+	display: none;
+	position: absolute;
+	padding: 12px 16px;
+	z-index: 1;
+	top: 50px;
+	left: -130px;
+	color: #222;
+	background-color: #fff;
+	border-radius: 3px;
+	filter: drop-shadow(0 1px 3px rgba(77, 77, 77, 0.5));
+	min-width: 150px;
+}
+
+.userInfo:hover .userInfo-content {
+	display: block;
+}
+
+.document-title {
+	justify-content: space-between;
+}
+
+#user-avt {
+	width: 35px;
+	height: 35px;
+	border-radius: 100%;
+}
+
+.userInfo-content::after {
+	border: 10px solid transparent;
+	border-bottom-color: var(--color-main-background);
+	bottom: 100%;
+	content: ' ';
+	height: 0;
+	width: 0;
+	position: absolute;
+	pointer-events: none;
+	color: #f9f9f9;
+	right: 25px;
+}
+
+.user-item-wrapper {
+	display: flex;
+	justify-content: flex-start;
+	align-items: center;
+}
+
+.user-item {
```

Comment:
Please add somea dditional styling to make the spacing fit the existing popover entries and the font also match the default `font-family: var(--loleaflet-font);`

See screenshot for comparison:
![image](https://user-images.githubusercontent.com/3404133/140533909-f1f763ae-211f-41d2-aa5c-b42b26b01559.png)